uuid.go

```diff
@@ -156,6 +158,53 @@ func (u UUID) String() string {
 	return string(buf)
 }
 
+// Format implements fmt.Formatter for UUID values.
+//
+// The behavior is as follows:
+// The 'x' and 'X' verbs output only the hex digits of the UUID, using a-f for 'x' and A-F for 'X'.
+// The 'v', 's' and 'q' verbs returns the canonical RFC-4122 string representation.
+// All other verbs not handled directly by the fmt package (like %p) return an empty string.
+func (u UUID) Format(f fmt.State, c rune) {
+	switch c {
+	case 'x', 'X':
+		s := hex.EncodeToString(u.Bytes())
+		if c == 'X' {
+			s = strings.Map(toCapitalHexDigits, s)
+		}
+		_, _ = io.WriteString(f, s)
+	case 'v', 's':
```

Comment:
How would you feel about scope-creeping a bit here, and adding a `%S` which returns the RFC format in all capital letters? Would there be any objections for this?